html/images/lang/en/labels/real-california-milk-90x90.png

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr" class="client-nojs">
+<head>
+<title>File:Real-california-milk-90x90.png - Open Food Facts EN</title>
```

Comment:
This seems to be HTML from Wikipedia.

Reply:
fixed. the proper logo was already commited.